packages/sdk-core/src/constants.ts

```diff
@@ -44,7 +44,9 @@ const getResolverData = (chainId: number): IResolverData => {
             nativeTokenSymbol: "",
         };
     return {
-        subgraphAPIEndpoint: `${baseUrl}-${networkData?.shortName}`,
+        subgraphAPIEndpoint: envReleaseTag
+            ? `${baseUrl}-${networkData?.name}`
+            : networkData.subgraphV1.hostedEndpoint,
```

Comment:
we use `.hostedEndpoint` outside of tests w/ a `SUBGRAPH_RELEASE_TAG` env variable set